src/lib/Scenes/Collection/Screens/__tests__/__snapshots__/CollectionHeader-tests.tsx.snap

```diff
@@ -1,64 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`CollectionHeader Renders collection header correctly 1`] = `
-Array [
-  <View
-    mb={2}
-    style={
-      Array [
-        Object {
-          "marginBottom": 8,
-        },
-      ]
-    }
-  >
-    <AROpaqueImageView
-      height={204}
-      imageURL="http://imageuploadedbymarketingteam.jpg"
-      width={750}
-    />
-  </View>,
-  <Text
-    color="#000"
-    fontFamily="ReactNativeAGaramondPro-Regular"
-    fontSize="32px"
-    lineHeight="38px"
-    ml={2}
-    style={
-      Array [
-        Object {
-          "color": "#000",
-          "fontFamily": "ReactNativeAGaramondPro-Regular",
-          "fontSize": 32,
-          "lineHeight": 38,
-          "marginLeft": 8,
-        },
-        Object {},
-      ]
-    }
-  >
-    Street Art Now
-  </Text>,
-  <Text
-    color="#000"
-    fontFamily="ReactNativeAGaramondPro-Regular"
-    fontSize="18px"
-    lineHeight="26px"
-    mt={2}
-    style={
-      Array [
-        Object {
-          "color": "#000",
-          "fontFamily": "ReactNativeAGaramondPro-Regular",
-          "fontSize": 18,
-          "lineHeight": 26,
-          "marginTop": 8,
-        },
-        Object {},
-      ]
-    }
-  >
-    &lt;p&gt;From &lt;a href="https://www.someartsyimage.net/artist/jean-michel-basquiat"&gt;Jean-Michel Basquiat&lt;/a&gt; to &lt;a href="https://www.someartsyimage.net/artist/shepard-fairey"&gt;Shepard Fairey&lt;/a&gt;, street artists are known for their creative expressions of rebellion.&lt;/p&gt;
-  </Text>,
-]
-`;
+exports[`renders properly 1`] = `"<view mb=\\"2\\"><aropaqueimageview imageurl=\\"http://imageuploadedbymarketingteam.jpg\\" height=\\"204\\" width=\\"750\\"></aropaqueimageview></view>"`;
```

Comment:
This looks different than before, but I still don't think it's capturing what you intended it to capture.